lab/torch/random.py

```diff
@@ -22,17 +22,25 @@ def global_random_state(_: TorchDType):
     else:
         parts = B.ActiveDevice.active_name.lower().split(":")
```

Comment:
```suggestion
        parts = B.ActiveDevice.active_name.lower().split(":", 1)
```

Not related to your PR/MPS, but a more safe way to ensure that there are never more than two parts.